### PR TITLE
Added the first 6 questions

### PR DESCRIPTION
**Pull request by rhys-wootton-cackett**
It bugs me that not all of it can have java syntax highlighting, but oh well, nothing I think you can do about that lol

### CODE REVIEW

**Comment by comp500** [comment]
Yeah, pygments doesn't allow you to use multiple languages at once.

**Comment by rhys-wootton-cackett** [comment]
Trash.

**Comment by comp500** [comment]
To be fair, they'd have to rewrite most of their code to support it, unless you made a custom language that was just the combination of both java and diff highlighting. It's also hard to make sure that valid java code (that isn't diff) doesn't get treated as diffs, unless you use strict Unified Diff format.

File: docs/wikibook.md [hunk 1]
```diff
@@ -61,3 +62,244 @@ void playGame(ArrayList<Character> characters, ArrayList<Item> items, ArrayList<
...
+
+```diff
+void playGame(ArrayList<Character> characters, ArrayList<Item> items, ArrayList<Place> places) {
+  [']
+  [']
+  [']
++        case "help":
++           Console.writeLine();
++           for (String cmd : HELP_COMMAND_STRINGS) {
++          	  Console.writeLine(cmd);
```
Comment (comp500) [inline docs/wikibook.md]: Would it be easier/better to keep the command strings inline (no for loop), as they aren't used elsewhere?

Comment (rhys-wootton-cackett) [reply docs/wikibook.md]: Probably, but I wrote this at god knows what time and it works, and doing it another way might confuse people lol.

**Comment by rhys-wootton-cackett** [comment]
Ah alright, well I'll just have new functions have the Java syntax highlighting, and then just leave changes within functions that already exist use diff. Beggars can't be choosers I guess.

File: docs/wikibook.md [hunk 2]
```diff
@@ -61,3 +62,244 @@ void playGame(ArrayList<Character> characters, ArrayList<Item> items, ArrayList<
...
+```java
+boolean saveGame(String filename, ArrayList<Character> characters, ArrayList<Item> items, ArrayList<Place> places) {
+   int count;
+   try {
+      FileOutputStream binaryWriter = new FileOutputStream(filename);
+      DataOutputStream writer = new DataOutputStream(binaryWriter);
+      writer.writeInt(characters.size());
+      
+      for (count = 0; count < characters.size() ; count++) {
+      	writer.writeInt(characters.get(count).id);
```
Comment (comp500) [inline docs/wikibook.md]: Inconsistent indentation - I'll fix this once I've merged it.

Comment (rhys-wootton-cackett) [reply docs/wikibook.md]: I do love Eclipse sometimes.

**Pull request merged**